homeassistant/components/forked_daapd/browse_media.py

```diff
@@ -229,6 +229,10 @@ def create_browse_media_response(
     if not children:  # Directory searches will pass in subdirectories as children
         children = []
     for item in result:
+        if item.get("data_kind") == "spotify" or (
```

Comment:
Shouldn't we only exclude Spotify if the user has configured Spotify in HA?

Reply:
I think it's better to exclude it altogether at least for now. The media browser is broken/very slow when trying to show Spotify content through the component. This may be due to the double proxying of artwork from Spotify -> Owntone -> HA -> frontend and also a suboptimal concurrency model from the Owntone server (it seems to use Apache/httpd). I just tried working around the latter by using a semaphore for the art fetching which improved things slightly but not enough to be useable. The media browsing proxy in HA is itself pretty inefficient, and if it can be avoided by going straight to the source the experience is much better. I think it makes much more sense to use the Spotify component itself for browsing.